src/ArduinoIoTCloud.cpp

```diff
@@ -17,444 +17,117 @@
-          execCloudEventCallback(_on_connect_event_callback, 0 /* callback_arg */);
-          CloudSerial.begin(9600);
-          CloudSerial.println("Hello from Cloud Serial!");
-        }
-      }
-      break;
-    case ArduinoIoTConnectionStatus::CONNECTING: {
-        int const ret_code_connect = connect();
-        Debug.print(DBG_VERBOSE, "ArduinoCloud.connect(): %d", ret_code_connect);
-        if (ret_code_connect == CONNECT_SUCCESS) {
-          _iotStatus = ArduinoIoTConnectionStatus::CONNECTED;
-          printConnectionStatus(_iotStatus);
-          execCloudEventCallback(_on_connect_event_callback, 0 /* callback_arg */);
-          CloudSerial.begin(9600);
-          CloudSerial.println("Hello from Cloud Serial!");
-        } else if (ret_code_connect == CONNECT_FAILURE_SUBSCRIBE) {
-          Debug.print(DBG_INFO, "ERROR - Please verify your THING ID");
-        }
-      }
-      break;
-  }
+ArduinoCloudProperty& ArduinoIoTCloudClass::addPropertyReal(String& property, String name, Permission const permission) {
+  return addPropertyReal(property, name, -1, permission);
 }
 
-void ArduinoIoTCloudClass::printDebugInfo() {
-  Debug.print(DBG_INFO, "***** Arduino IoT Cloud - configuration info *****");
-  Debug.print(DBG_INFO, "Device ID: %s", getDeviceId().c_str());
-  Debug.print(DBG_INFO, "Thing ID: %s", getThingId().c_str());
-  Debug.print(DBG_INFO, "MQTT Broker: %s:%d", _brokerAddress.c_str(), _brokerPort);
+ArduinoCloudProperty& ArduinoIoTCloudClass::addPropertyReal(String& property, String name, int tag, Permission const permission) {
+  ArduinoCloudProperty* p = new CloudWrapperString(property);
+  return Thing.addPropertyReal(*p, name, permission, tag);
 }
 
 void ArduinoIoTCloudClass::addCallback(ArduinoIoTCloudEvent const event, OnCloudEventCallback callback) {
   switch (event) {
-    case ArduinoIoTCloudEvent::SYNC:       _on_sync_event_callback       = callback; break;
-    case ArduinoIoTCloudEvent::CONNECT:    _on_connect_event_callback    = callback; break;
+    case ArduinoIoTCloudEvent::SYNC:       _on_sync_event_callback = callback; break;
+    case ArduinoIoTCloudEvent::CONNECT:    _on_connect_event_callback = callback; break;
     case ArduinoIoTCloudEvent::DISCONNECT: _on_disconnect_event_callback = callback; break;
   }
-}
+};
 
-void ArduinoIoTCloudClass::execCloudEventCallback(OnCloudEventCallback & callback, void * callback_arg) {
+void ArduinoIoTCloudClass::execCloudEventCallback(OnCloudEventCallback& callback, void* callback_arg) {
   if (callback) {
     (*callback)(callback_arg);
   }
 }
-
 void ArduinoIoTCloudClass::printConnectionStatus(ArduinoIoTConnectionStatus status) {
   switch (status) {
-    case ArduinoIoTConnectionStatus::IDLE:         Debug.print(DBG_INFO,  "Arduino IoT Cloud Connection status: IDLE");         break;
+    case ArduinoIoTConnectionStatus::IDLE:         Debug.print(DBG_INFO, "Arduino IoT Cloud Connection status: IDLE");         break;
```

Comment:
This is a unnecessary formatting change which enlarges the change set that needs to be reviewed without any functional change. Please refrain from formatting changes when doing functional changes and restore the original version.

Reply:
fixed